nemo/export/tensorrt_llm.py

```diff
@@ -316,12 +318,24 @@ def export(
                     model_type = get_model_type(nemo_checkpoint_path)
 
                 if model_type is None:
-                    raise Exception("model_type needs to be specified, got None.")
+                    raise Exception(
+                        "Parameter model_type needs to be provided and cannot be inferred from the checkpoint. "
+                        "Please specify it explicitely."
+                    )
 
                 if model_type not in self.get_supported_models_list:
                     raise Exception(
```

Comment:
In Python they recommend not to use the `Exception` class and try something more specific. Here and in other such places in the function `ValueError` would look prettier.

Reply:
I used `Exception` for consistency with the rest of the code but you're right, at some point we should turn these generic exceptions to sth more explicit